QtAwesome/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+project(QtAwesome)
+
+add_library(${PROJECT_NAME}
```

Comment:
Looks like library cannot build as shared on Windows. What about to add CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS or throw fatal error for such case or force static lib as qmake project?